chainer/training/extensions/parameter_statistics.py

```diff
@@ -65,7 +68,7 @@ def __init__(self, links, statistics=default_statistics,
             links = links,
         self._links = links
 
-        self._statistics = statistics
+        self._statistics = statistics or {}
```

Comment:
Please use `is None` when you check `None`. See also [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).

Reply:
Thanks, fixed!